validation/validation.go

```diff
@@ -1,27 +1,41 @@
 package validation
 
 import (
+	"errors"
 	"fmt"
 	"net/url"
 
 	"github.com/fatih/color"
 	"github.com/preslavmihaylov/todocheck/config"
+	"github.com/preslavmihaylov/todocheck/fetcher"
+	"github.com/preslavmihaylov/todocheck/issuetracker"
 )
 
 // Validate validates the values of given configuration
 func Validate(cfg *config.Local) []error {
-	var errors []error
+	var errs []error
 
 	if err := validateIssueTracker(cfg); err != nil {
-		errors = append(errors, err)
+		errs = append(errs, err)
 	}
 
 	if err := validateAuthOfflineURL(cfg); err != nil {
-		errors = append(errors, err)
+		errs = append(errs, err)
 	}
 
 	if err := validateIssueTrackerOrigin(cfg); err != nil {
-		errors = append(errors, err)
+		errs = append(errs, err)
+	}
+
+	url, err := issuetracker.HealthcheckURL(cfg.IssueTracker, cfg.Origin)
+	if err != nil {
+		if !errors.Is(err, issuetracker.ErrUnsupportedHealthCheck) {
+			errs = append(errs, err)
+		}
+	} else {
+		if !fetcher.IsHealthy(cfg.IssueTracker, url) {
+			errs = append(errs, fmt.Errorf("repository %s not found. Is the repository private? More info: https://github.com/preslavmihaylov/todocheck#github", url))
+		}
```

Comment:
Final touch.

For consistency with the rest of this function, could you encapsulate your new validation logic into a `validateIssueTrackerExists(cfg)`